lib/api/api.dart

```diff
@@ -520,6 +523,29 @@ class _APIRequests {
     await _actionExecution(APIAction.GET_HOMEWORK);
     http.postToAPI("/homework/v1/report/$id", {}, jsonEncode({}), _api._authenticationUser.getJWT());
   }
+
+  Future<bool> sendKrankmeldung(String sek, String name, String grade, String klasse, String leader, String email, String remarks) async {
```

Comment:
Ich weiß das wir das nicht wirklich fest gelegt haben bis jetzt, aber ich persönlich fände es schöner eine Klasse Krankmeldung in der models Datei zu erstellen und ein Objekt davon hier zu übergeben.
Da muss dann aber eine toJson Methode rein